src/FileProcessor/TypoScript/TypoScriptFileProcessor.php

```diff
@@ -143,36 +141,28 @@ private function processFile(File $file): void
                 return;
             }
 
-            $editorConfigConfigurationBuilder = new EditorConfigConfigurationBuilder();
-            $editorConfigConfigurationBuilder->withIndent(Indent::createSpaceWithSize(4));
-
-            $editorConfiguration = $this->editorConfigParser->extractConfigurationForFile(
-                $file,
-                $editorConfigConfigurationBuilder
-            );
+            // keep original json format
+            $tabMatches = Strings::match($file->getFileContent(), "#^\n#");
+            $indentStyle = $tabMatches ? 'tab' : 'space';
 
             $prettyPrinterConfiguration = PrettyPrinterConfiguration::create();
             $prettyPrinterConfiguration = $prettyPrinterConfiguration->withEmptyLineBreaks();
 
-            if ('tab' === $editorConfiguration->getIndentStyle()) {
+            if ('tab' === $indentStyle) {
                 $prettyPrinterConfiguration = $prettyPrinterConfiguration->withTabs();
             } else {
-                $prettyPrinterConfiguration = $prettyPrinterConfiguration->withSpaceIndentation(
-                    $editorConfiguration->getIndentSize()
-                );
+                // default indent
+                $prettyPrinterConfiguration = $prettyPrinterConfiguration->withSpaceIndentation(4);
             }
 
             $prettyPrinterConfiguration = $prettyPrinterConfiguration->withClosingGlobalStatement();
-
             $this->typoscriptPrinter->setPrettyPrinterConfiguration($prettyPrinterConfiguration);
 
             $printStatements = $this->filterRemovedStatements($originalStatements, $file);
-
             $this->typoscriptPrinter->printStatements($printStatements, $this->output);
 
             $newTypoScriptContent = $this->applyTypoScriptPostRectors($this->output->fetch());
-
-            $typoScriptContent = rtrim($newTypoScriptContent) . $editorConfiguration->getNewLine();
+            $typoScriptContent = rtrim($newTypoScriptContent) . "\n";
```

Comment:
This might require better endline resolution in the future. But having the most tools, even php-parser this endline as hardcoded default, I think we're ok with json as well: https://github.com/nikic/PHP-Parser/blob/678ccbe0720549c51d30b76d094a117eac819e9e/lib/PhpParser/PrettyPrinterAbstract.php#L159